src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+    margin-right: 50px;
+  }
+}
```

Comment:
Nav links color is not black any more. That's why the header tests don't pass.

Reply:
thank you!you helped me a lot